cmd/frontend/graphqlbackend/search.go

```diff
@@ -55,19 +51,23 @@ func maxReposToSearch() int {
 	}
 }
 
-// Search provides search results and suggestions.
-func (r *schemaResolver) Search(args *struct {
+type searchArgs struct {
 	Version     string
 	PatternType *string
 	Query       string
 	After       *graphql.ID
 	First       *int32
-}) (interface {
+}
+
+type searchIntf interface {
```

Comment:
Unless there is a naming conflict, I think this is more Go idiomatic:

```suggestion
type searchResolver interface {
```

Reply:
Can't: https://sourcegraph.com/github.com/sourcegraph/sourcegraph@4302f2bd8542ecf733ec8d37d31e4b80e9c21c90/-/blob/cmd/frontend/graphqlbackend/search.go#L158

Reply:
Though if you have an alternative idiomatic suggestion, LMK.

Reply:
How's `searchResponser`?

Reply:
Or `searchResulter`? `searchIntf` is certainly not Go-ey

Reply:
See #6170.